scripts/remote-provision.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+
+set -o errexit   # abort on nonzero exit status
+set -o errtrace  # pass ERR trap down to functions, substitutions, etc
+set -o nounset   # abort on unbound variable
+set -o pipefail  # don’t hide errors within pipes
+
+[[ "${BASH_VERSINFO:-0}" -ge 4 ]] || { echo "Bash version 4 or higher is required."; exit 1; }
+
+usage() {
+    echo "Usage $0 [OPTIONS] <teleport-tunnel|ipv4>
+
+    Options:
+        -r, --reprovision   Re-provisioning the device and skips fetching of attestation certificate
```

Comment:
In case the `usr/persistent/se/keystore` does not exist, the script should exit. I think that this is enough of a test to ensure that the attestation certificate is not going to get wiped on unprovisioned orbs.

Reply:
I don't see connection. `/usr/persistent/se/keystore` even if this dir exists `-r` seems very intrusive, we will preserve att.token, but new certs will be generated that later needs to be uploaded.

I would still prompt user and add `-f/force` flag for disabling prompt.